### ISSUE TITLE
No single method: _setup of interface: cljs.repl.IJavaScriptEnv found for function

### ISSUE THREAD

**Issue by pate** [Day8/re-com#40]
Added `[re-com "0.5.4"]` dependency and `boot dev` throws this:

```
clojure.lang.Compiler$CompilerException: java.lang.IllegalArgumentException: No single method: _setup of interface: cljs.repl.IJavaScriptEnv found for function: -setup of protocol: IJavaScriptEnv, compiling:(cemerick/piggieback.clj:149:5)
     java.lang.IllegalArgumentException: No single method: _setup of interface: cljs.repl.IJavaScriptEnv found for function: -setup of protocol: IJavaScriptEnv
```

Feels like a weird dependency conflict. Here are my other dependencies:

```
[org.clojure/clojurescript "0.0-3308" :scope "provided"]
[reagent "0.5.0"]
[re-frame "0.4.1"]
[re-com "0.5.4"]
[matchbox "0.0.7-SNAPSHOT"]
[secretary "1.2.3"]]
```

**Comment by mike-thompson-day8** [comment]
I saw someone report an issue with boot and re-com ... and it turned out to be something around the version of clj-time used by re-com.   It is just a vague memory from last week or the week before.  Can't remember anything else.

**Comment by mike-thompson-day8** [comment]
We don't use boot, yet,  but I'm told this change will fix it 0333f71fb43b6bc7f453c6ce138630c332d745a5
